src/zope/app/wsgi/README.txt

```diff
@@ -143,7 +143,7 @@ example:
   >>> import os, tempfile
   >>> temp_dir = tempfile.mkdtemp()
   >>> sitezcml = os.path.join(temp_dir, 'site.zcml')
-  >>> written = open(sitezcml, 'w').write('<configure />')
+  >>> with open(sitezcml, 'w') as f: _ = f.write('<configure />')
```

Comment:
It would be easier to read when breaking this to two lines. Maybe a function could hide the implementation details of writing to the temporary file.